proposals/nnnn-structured-concurrency.md

```diff
@@ -39,7 +39,9 @@ Each step in our dinner preparation is an asynchronous operation, so there are n
 
 However, even though our dinner preparation is asynchronous, it is still *sequential*. It waits until the vegetables have been chopped before starting to marinate the meat, then waits again until the meat is ready before preheating the oven. Our hungry patrons will be very hungry indeed by the time dinner is finally done.
 
-To make dinner preparation go faster, we want to use concurrency. The vegetables can be chopped at the same time as the meat is marinating and the oven is preheating. We can be combining the ingredients into the dish while the oven preheats. By overlapping the steps, we can cook our dinner faster.
+To make dinner preparation go faster, we need to perform some of the tasks in *parallel*, but at the same time, we need to do so with some form of structure. Not all tasks can be just launched in parallel, hoping for the best. In order to properly deadl with parallelism, we need structure, and that structure is *concurrency*. The vegetables can be chopped at the same time as the meat is marinating and the oven is preheating. We can be combining the ingredients into the dish while the oven preheats. But we cannot prepare the dish before it's individual parts (the veggies and meat) are prepared. 
```

Comment:
I realize this made it a bit longer... but it's important that parallelism != concurrency.
If this is too long I can try to figure out a shorter phrasing.

It is tremendously important to detach the structure of things (concurrency) from its execution (parallelism), by providing good concurrency tools/language/concepts we enable parallelism. 

The amount of parallelism is flexible -- there could be no parallelism whatsoever in the above example, we could be running all things on 1 thread but it's still concurrent. The detachment of thinking about specifics of the parallelism is important and powerful to keep in mind; even if we just hint a bit at it here. wdyt?

Reply:
I like this better, thank you